vagrant/scripts.sh

```diff
@@ -147,20 +147,19 @@ setupdocker ()
         sudo apt-get install -y docker-ce
         sudo gpasswd -a ubuntu docker
         sudo sed -i -e 's,ExecStart=/usr/bin/dockerd -H fd://,ExecStart=/usr/bin/dockerd,' /lib/systemd/system/docker.service
-    sudo sh -c 'cat <<EOF > /etc/docker/daemon.json
-{
-    "hosts": ["unix:///var/run/docker.sock", "tcp://0.0.0.0:2375", "fd://"]
-}
-EOF'
     elif [ $DISTRO == Fedora ]; then
         sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
         sudo dnf -y install docker-ce
         sudo gpasswd -a vagrant docker
         sudo firewall-cmd --permanent --add-port=2375/tcp
```

Comment:
If there is a possibility to parametrize port, will be good.

Reply:
We use value 2375 in many other places, e.g. test framework configuration files to run tests. This action is done just once on a target machine, so if it is necessary to change docker daemon port, the user might as well edit this file once.